src/app/admin-panel/teacher-connection/snackbar-mess/snackbar-mess.component.html

```diff
@@ -0,0 +1,3 @@
+<span class="choise-done">
+  Вибір зроблено!!! 
```

Comment:
please, clarify wording that is used across the application

Reply:
resolved